editoast/editoast_derive/src/modelv2/args.rs

```diff
@@ -68,4 +68,12 @@ impl GeneratedTypeArgs {
     pub fn ident(&self) -> syn::Ident {
         syn::Ident::new(self.type_name.as_ref().unwrap(), Span::call_site())
     }
+
+    pub fn visisbililty(&self) -> syn::Visibility {
```

Comment:
Let's fix the typos (2 extra letters) at the same type (unless it's on purpose 🤔 but then I do not understand why).

```suggestion
    pub fn visibility(&self) -> syn::Visibility {
```

Reply:
<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExdXhtNDduNHMzMjZhMW90MXpqZTB4cjV5djF2Y2NsaHRtYXJpMHEzNCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/yBPWhZIatOqJO/giphy.gif"/>